### DISCUSSION TITLE
cheatsheets/todocli/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/cheatsheets#9] [General]
# cheatsheets/todocli/

https://shanenull.com/cheatsheets/todocli/

**Comment by shane0** [comment]
```sh
#!/usr/bin/env bash
pandoc -t revealjs -s -o ./slides/daily.html ./slides/daily.md -V revealjs-url=https://unpkg.com/reveal.js@3.9.2/ -V theme=solarized
cp ./slides/daily.html ./docs/
```

**Comment by shane0** [comment]
https://shanenull.com/cheatsheets/django/

**Comment by shane0** [comment]
try https://github.com/kmonad/kmonad

revive the keyboards/ repo

**Comment by shane0** [comment]
Laptop parts projects
https://youtu.be/WLP_L7Mgz6M?si=Tf5JS0VA9CbdriW1

**Comment by shane0** [comment]
# whatsapp

- my response when people invite me to whatsapp
- https://www.eff.org/deeplinks/2016/10/where-whatsapp-went-wrong-effs-four-biggest-security-concerns

**Comment by shane0** [comment]
The standard deviation \( \sigma \) of a dataset is calculated using the following steps:

1. Calculate the mean (\( \mu \)) of the dataset by adding up all the values and dividing by the total number of values.
   
   \[ \mu = \frac{1}{N} \sum_{i=1}^{N} x_i \]

   Where:
   - \( N \) is the total number of values in the dataset.
   - \( x_i \) represents each individual value in the dataset.

2. Calculate the squared difference between each value (\( x_i \)) and the mean (\( \mu \)).

   \[ (x_i - \mu)^2 \]

3. Calculate the average of these squared differences, which is called the variance (\( \sigma^2 \)).

   \[ \sigma^2 = \frac{1}{N} \sum_{i=1}^{N} (x_i - \mu)^2 \]

4. Finally, take the square root of the variance to get the standard deviation (\( \sigma \)).

   \[ \sigma = \sqrt{\sigma^2} \]

The standard deviation gives us a measure of the dispersion or spread of the values in the dataset around the mean. It helps in understanding the variability or consistency of the data points.
